tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -235,4 +237,231 @@ describe("E2E Testing for taqueria flextesa plugin",  () => {
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts/hello-tacos-two.tz`);
+
+            // 2. Run taq typecheck
+            const {stdout, stderr} = await exec(`taq typecheck ${dockerName}`, {cwd: `./${taqueriaProjectPath}`});
+
+            // 3. Verify that both are well-typed and contain no errors
+            expect(stdout).toBe(contents.twoRowTable);
+            expect(stderr).toBe("");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    });
+
+    test('Verify that taqueria typechecker task can typecheck multiple (but not all) contracts using typecheck [sourceFile1] [sourceFile2] command', async () => {
+        try {
+            // 1. Copy two contracts from data folder to /contracts folder under taqueria project
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts/hello-tacos-one.tz`);
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts/hello-tacos-two.tz`);
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts/hello-tacos-three.tz`);
+
+            // 2. Run taq typecheck hello-tacos-one.tz hello-tacos-two.tz
+            const {stdout, stderr} = await exec(`taq typecheck ${dockerName} hello-tacos-one.tz hello-tacos-two.tz`, {cwd: `./${taqueriaProjectPath}`});
+
+            // 3. Verify that both are well-typed and contain no errors
+            expect(stdout).toBe(contents.twoRowTable);
+            expect(stderr).toBe("");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    });
+
+    test('Verify that taqueria typechecker task will display proper message if user tries to typecheck contract that does not exist', async () => {
+        try {
+            // 1. Run taq typecheck ${contractName} for contract that does not exist
+            const {stdout, stderr} = await exec(`taq typecheck ${dockerName} test.tz`, {cwd: `./${taqueriaProjectPath}`})
+
+            // 2. Verify that output includes a table and an error message
+            expect(stdout).toBe(contents.nonExistent)
+            expect(stderr).toContain("Does not exist");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    });
+
+    test("Verify that taqueria typechecker task emits error and yet displays table if contract is ill-typed", async () => {
+        try {
+            // 1. Copy contract from data folder to taqueria project folder
+            execSync(`cp e2e/data/hello-tacos-ill-typed.tz ${taqueriaProjectPath}/contracts`)
+
+            // 2. Run taq typecheck hello-tacos-ill-typed.tz
+            const {stdout, stderr} = await exec(`taq typecheck ${dockerName} hello-tacos-ill-typed.tz`, {cwd: `./${taqueriaProjectPath}`})
+
+            // 3. Verify that output includes a table and an error message
+            expect(stdout).toBe(contents.typeError)
```

Comment:
Same as the above for the double output

Reply:
This is different from the above. The output looks like this, which looks fine:

```
jimmychenchen@Jimmys-MBP taco % taq tc local hello-tacos-ill-typed.tz
 
Ill typed contract:
  01: { parameter nat ;
  02:   storage string ;
  03:   code { UNPAIR ;
  04:          SWAP ;
  05:          DUP ;
  06:          DUG 2 ;
  07:          SWAP ;
  08:          DUP ;
  09:          DUG 2 ;
  10:          COMPARE ;
  11:          GT ;
  12:          IF { DROP 2 ; PUSH string "NOT_ENOUGH_TACOS" ; FAILWITH }
  13:             { SWAP ; SUB ; ABS ; NIL operation ; PAIR } } }
  14: 
  15: 
At line 10 characters 9 to 16, wrong stack type for instruction COMPARE:
  [ nat @parameter : string @storage : ... ].
Bad stack item 1.
Type nat is not compatible with type string.
Fatal error:
  ill-typed script

┌──────────────────────────┬─────────┐
│ Contract                 │ Result  │
├──────────────────────────┼─────────┤
│ hello-tacos-ill-typed.tz │ Invalid │
└──────────────────────────┴─────────┘

```

Reply:
👍